packages/zosuss/src/Shell.ts

```diff
@@ -164,6 +171,14 @@ export class Shell {
         return this.executeSsh(session, cwdCommand, stdoutHandler);
     }
 
+    public static async isConnectionValid(session: SshSession): Promise<boolean>{
```

Comment:
Can we have a system test that tests this please? :)

Reply:
see e95f9fd41951ca5f1968fca5b6ad167c4816f755